test/unit/test_probe.cpp

```diff
@@ -472,28 +1099,59 @@ TEST(probe, gpu_v_i) {
     }
 }
 
+TEST(probe, gpu_v_cell) {
+    context ctx = make_context(proc_allocation{1, arbenv::default_gpu()});
+    if (has_gpu(ctx)) {
+        run_v_cell_probe_test<gpu::backend>(ctx);
```

Comment:
Well, that answers my question about whether the GPU backend was being tested.

Reply:
Ah, yes :)